mocks.pkg/roadmap

```diff
@@ -0,0 +1,10 @@
+- fill methods args cfgs for chains (eth)
+- ask jintao to cr8 a package for sdk
+- add rpc nods url to ENV
+- add generation and deploy in github actions
+
+improvments:
```

Comment:
do we want to make an md or docs for this one or remove it

Reply:
ye, it`s a quick notes, i`ll transfer all of that